README.md

```diff
@@ -25,6 +25,7 @@ yarn add typesettings-js
   - [generate()](#generate())
   - [generateFonts()](#generateFonts())
   - [generateFontFace()](#generateFontFace())
+- [Typescript](#typescript)
```

Comment:
nit: Indentation

Reply:
indent is correct. It's not meant to be under `API`